frontend/assets/css/fonts.scss

```
@@ -277,6 +293,27 @@ h1 {
   font-weight: var(--button_font_weight);
 }
 
+
 .button_text{
   @include button_text
 }
+
+@mixin toast_text(){
+  font-family:var(--mukta-family);
+  font-size:var(--standard_text_font_size);
+  font-weight: var(--mukta-semi-bold);
+}
+
+.toast_text{
+  @include toast_text
+}
+
+@mixin small_toast_text(){
+  font-family:var(--mukta-family);
+  font-size: var(--small_text_font_size);
+  font-weight: var(--mukta-light);
+}
```

nitpick: I am a huge fan of [tailwind](https://tailwindcss.com/docs/font-size) and like their conventions,
e.g. this could be a version
```suggestion
@mixin text-toast--sm(){
  font-family:var(--mukta-family);
  font-size: var(--text-font-size--sm);
  font-weight: var(--mukta-light);
}
```

---

Also big tailwind fan over here, i tried to follow the naming in the project.